src/search/operator_counting/operator_counting_heuristic.cc

```diff
@@ -24,10 +24,11 @@ OperatorCountingHeuristic::OperatorCountingHeuristic(const Options &opts)
         variables.push_back(lp::LPVariable(0, infinity, op_cost));
     }
     named_vector::NamedVector<lp::LPConstraint> constraints;
+    lp::LinearProgram lp(lp::LPObjectiveSense::MINIMIZE, move(variables), move(constraints), infinity);
     for (const auto &generator : constraint_generators) {
-        generator->initialize_constraints(task, constraints, infinity);
+        generator->initialize_constraints(task, lp);
     }
-    lp_solver.load_problem(lp::LinearProgram(lp::LPObjectiveSense::MINIMIZE, move(variables), move(constraints)));
+    lp_solver.load_problem(move(lp));
```

Comment:
I don't think the move has an effect since `load_problem` accepts a const reference.